src/components/navigation/Login.vue

```diff
@@ -91,11 +131,48 @@ export default Vue.extend({
         },
         forgotPassword() {
             this.$root.$emit('bv::show::modal', 'passwordModal');
-        }
+        },
+        register() {
+        this.$root.$emit('bv::show::modal', 'registerModal');
+    }
     }
 });
 </script>
 
-<style scoped>
 
+<style lang="scss">
+.title-header {
+    font-family: AvenirLTStd-Light;
```

Comment:
Maybe some of these directives would be better stored as variables, so that if we do need to make a change in the future they are in a centralized place?